app/controllers/decidim/participatory_documents/admin/suggestions_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryDocuments
+    module Admin
+      class SuggestionsController < Admin::ApplicationController
+        include Decidim::Admin::Filterable
+        include Decidim::Admin::Paginable
+        helper Decidim::ParticipatoryDocuments::Admin::SuggestionHelper
+
+        helper_method :suggestions, :suggestion, :document
+
+        def show
+          enforce_permission_to :update, :participatory_document, suggestion: suggestion
+          @form = form(Decidim::ParticipatoryDocuments::Admin::AnswerSuggestionForm).from_model(suggestion)
+        end
+
+        private
+
+        def search_field_predicate
+          :body_cont
+        end
+
+        def filters_with_values
+          {
+            state_eq: suggestion_stats
+          }
+        end
+
+        def suggestion_stats
+          Suggestion::POSSIBLE_STATES
+        end
+
+        def filters
+          [
+            :state_eq
+          ]
+        end
+
+        def suggestions
+          # filtered_collection
+          paginate(filtered_collection)
```

Comment:
I think that `filtered_collection` is already paginated.

![image](https://user-images.githubusercontent.com/1401520/212657198-ce2e2552-cbc7-40d5-8e78-5a7aa6a90a22.png)

Reply:
Yes, it is. We are not overriding filtered_collection, therefore, i have removed the pagination call.